bazel/cc_toolchains/clang_configuration.bzl

```diff
@@ -89,6 +93,21 @@ def _compute_mac_os_sysroot(repository_ctx):
     output = _run(repository_ctx, [xcrun, "--show-sdk-path"]).stdout
     return output.splitlines()[0]
 
+def _compute_bsd_sysroot(repository_ctx):
+    """Look around sysroot. Return root(/) if nothing found."""
+    cmd = repository_ctx.which("printenv")
+    readlink = repository_ctx.which("readlink")
+
+    # 'it-just-works' for Cmake users
+    var = "CMAKE_SYSROOT"
```

Comment:
Is this a freebsd-specific logic, or just trying to add some cmake consistency? Did you need this in order to make it work for yourself?

Note, if needed, you should probably be using repository_ctx functions instead of shelling out. e.g.:

```
sysroot = repository_ctx.os.environ.get("CMAKE_SYSROOT")
sysroot_path = repository_ctx.path(sysroot)
if sysroot_path.exists:
  return sysroot_path.realpath
return None
```

That way you can also write something that's more platform-independent.

Reply:
I agree. It is simpler in the way.